libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -109,17 +110,17 @@ uint8_t AC_Fence::get_enabled_fences() const
 }
 
 // additional checks for the polygon fence:
-bool AC_Fence::pre_arm_check_polygon(const char* &fail_msg) const
+bool AC_Fence::pre_arm_check_polygon(const char* &fail_msg)
 {
     if (!(_enabled_fences & AC_FENCE_TYPE_POLYGON)) {
         // not enabled; all good
         return true;
     }
 
-    if (!_boundary_valid) {
-        fail_msg = "Polygon boundary invalid";
-        return false;
-    }
+    // if (!_poly_loader.valid()) {
```

Comment:
We're going to make this behaviour change permanent.

Reply:
ok, I'd forgotten about this change.. should we just remove these lines then or should we have a valid() call that takes an instance?